src/mvContainers.h

```diff
@@ -160,6 +161,7 @@ struct mvWindowAppItemConfig
     bool             no_close = false;
     bool             no_background = false;
     bool             collapsed = false;
+    bool             unsaved_document = false;
```

Comment:
`unsaved_document` - This field is unused, the corresponding argument goes directly to `config.windowflags`.